### PR TITLE
#2073_Remove double header from light wallet recovery

### PR DESCRIPTION
**Pull request by AlexeyTuboltsev**
remove header and explanation from RegisterLightWallet component if it's recovery.
fix css for footnote.

### CODE REVIEW

File: app/components/wallet-selector/light/RegisterLightWallet.tsx [hunk 1]
```diff
@@ -120,20 +120,19 @@ export const RegisterWalletComponent: React.SFC<
...
-        <FormattedMessage id="wallet-selector.neuwallet.restore-prompt" />
-      ) : (
-        <FormattedMessage id="wallet-selector.neuwallet.register-prompt" />
-      )}
-    </h2>
-    <p className={styles.explanation}>
-      <FormattedMessage tagName="span" id="wallet-selector.neuwallet.explanation-1" />
-    </p>
-
+    {props.restore ? null : (
```
Comment (banciur) [inline app/components/wallet-selector/light/RegisterLightWallet.tsx]:
```suggestion
    {props.restore && (
      <>
        <h2
          className={cn(styles.title, "text-center mb-4")}
          data-test-id="modals.wallet-selector.register-restore-light-wallet.title"
        >
          <FormattedMessage id="wallet-selector.neuwallet.register-prompt" />
        </h2>
        <p className={styles.explanation}>
          <FormattedMessage tagName="span" id="wallet-selector.neuwallet.explanation-1" />
        </p>
      </>
    )}
```

Comment (banciur) [reply app/components/wallet-selector/light/RegisterLightWallet.tsx]: Isn't `&&` preferred way to render conditionally?

Comment (desfero) [reply app/components/wallet-selector/light/RegisterLightWallet.tsx]: hm, but still we need to negate here :)

Comment (banciur) [reply app/components/wallet-selector/light/RegisterLightWallet.tsx]: right :D

**Pull request merged**